matisse/src/main/java/com/zhihu/matisse/internal/utils/MediaStoreCompat.java

```diff
@@ -90,16 +93,26 @@ public void dispatchCaptureIntent(Context context, int requestCode) {
 
     private File createImageFile() throws IOException {
```

Comment:
Yes, since this annotation only contributes to code readability, so I didn't add that yet. Of course we can do it.